packages/design-system/CHANGELOG.md

```
@@ -4,6 +4,12 @@
 
 ### Patch Changes
 
+- Fix component tests after dependencies update.
+
+## 1.0.19
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content

(MD024, no-duplicate-heading)